### ISSUE TITLE
ARK server 0/0 player count, cant join.

### ISSUE THREAD

**Issue by Skeleton022** [GameServerManagers/LinuxGSM#2785]
## User Story

ARK server not working propery 0/0 player.

## Basic info
uname -a :Linux ark 4.15.0-91-generic #92-Ubuntu SMP Fri Feb 28 11:09:48 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux
Game: ARK
Tried to use ./arkserver postdetails as show on the website, but this is the result: Please share the following url for support:
(no url)

## Further Information
Validate: Success! App '376030' already up to date.
console:
Using binned.
4.5.1-0+UE4 7038 3077 404 10
[S_API FAIL] SteamAPI_Init() failed; SteamAPI_IsSteamRunning() failed.
Setting breakpad minidump AppID = 346110
Only changed the from 0.0.0.0 to 192.168.6.4 and that it, everything else is the default!
On steam in the server browser I see ARK:Survival Evolved 0/0, and can not connect to it!

I have tried to reinstall it 20 times, yesterday, and today as well. Tried with fully configured (as I would need) and also with default config, none of that works.
At first I thought it was some steam issue, but after a quick searchI found that it is only a warning.

Please help me to resolve this issue, I really like linuxgsm, I have used a lot in the past, and It worked back then.

## To Reproduce
Install the server. ->not working
## Expected behaviour
working

**Comment by Skeleton022** [comment]
So literally no one here knows what up with this situation? 

So I managed to get it working bí moving the server to a network with active dhcp server in it. (Idk why i helped... it had static ip soooooo...)

But now only I can see it on the local network, and nobody else can join. The steam server browser saying The server is not responding.

I did the port forward (27015/tcp, 7777/udp, 7778/udp just like back when it worked, but nothing)

Can someone do something with this issue?
